jboss-eap-standalone-openshift/README.md

```diff
@@ -0,0 +1,109 @@
+# Azure template for instantiating and deploying a standalone Red Hat JBoss EAP on OCP on Azure
+
+**NOTE: This template creates an ephemeral instance of OpenShift Container Platform and Red Hat JBoss EAP. You are responsible for backing up any data that you want to save while using this instance**
+
+1. Create a Single VM OpenShift deployment using the Azure Portal
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fjboss-eap-standalone-openshift%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fjboss-eap-standalone-openshift%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+
+------
+
+This template deploys OpenShift Container Platform on Azure.
+
+![Console](images/parameters.png)
+
+
+#### Subscription
+Accept the default subscription ID value. Note: this field is not shown in the picture above.
+
+#### Resource Group
+Select "Create new" resource group if one does not currently exist. Enter "ocp" or a name of your choosing in the input field.
+
+#### Location
+The geographic location in which to deploy OpenShift Container Platform.
+
+#### Admin User
+Supply a username which will be used for SSH access,OpenShift Container Platform web console, JBoss EAP Admin .
+
+#### Admin Password
+Supply a password which will be used for the Openshift Container Platform web console.
+
+#### Rhsm User
```

Comment:
add something to the description here about where to get these values, otherwise folks will think it's like any other deployment and work with whatever I provide

Reply:
done